hazelcast/src/test/java/com/hazelcast/client/impl/protocol/task/dynamicconfig/AddMapConfigMessageTaskTest.java

```diff
@@ -135,7 +135,8 @@ public void doNotThrowException_whenNullValuesProvidedForNullableFields() {
                 mapConfig.getMetadataPolicy().getId(),
                 mapConfig.isPerEntryStatsEnabled(),
                 mapConfig.getDataPersistenceConfig(),
-                mapConfig.getTieredStoreConfig()
+                mapConfig.getTieredStoreConfig(),
+                null
```

Comment:
it would be good to add test similar to `testDataPersistenceAndTieredStoreConfigTransmittedCorrectly` for the attribute partitioning strategy, maybe both with nulls and specific values.

Reply:
+

Reply:
Null value is checked in the test above actually and I've disabled paths for adding arrays with null (or empty) elements either way.